File: app/src/main/java/jogja/telkom/csrassistant/Adapter/Instruksi.java
package jogja.telkom.csrassistant.Adapter;

import java.io.Serializable;
import java.util.ArrayList;

public class Instruksi  implements Serializable {
        private String instruksi;
        private ArrayList<ChildInstruksi> childInstruksis;

        public String getInstruksi() {
            return instruksi;
        }

        public void setInstruksi(String instruksi) {
            this.instruksi = instruksi;
        }

        public ArrayList<ChildInstruksi> getChildDataItems() {
            return childInstruksis;
        }

        public void setChildInstruksis(ArrayList<ChildInstruksi> childInstruksis) {
            this.childInstruksis = childInstruksis;
        }
    }

File: app/src/main/java/jogja/telkom/csrassistant/Adapter/Instruksi.kt
package jogja.telkom.csrassistant.Adapter

import java.io.Serializable
import java.util.ArrayList

class Instruksi : Serializable {
    var instruksi: String? = null
    var childDataItems: ArrayList<ChildInstruksi>? = null
        private set

    fun setChildInstruksis(childInstruksis: ArrayList<ChildInstruksi>) {
        this.childDataItems = childInstruksis
    }
}
